examples/seed.yaml

```diff
@@ -82,14 +56,7 @@ tags:
 ---
 kind: Function
 name: http-py
-code: |
```

Comment:
Could we leave one of these as the function 'literal' to show that we support both?